custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js

```diff
@@ -626,12 +626,23 @@ function ServiceDeliveryDashboardController($rootScope, $scope, $http, $location
 
     }
 
+    function isAllRow(rowData) {
+        return (rowData['state_name'] === 'All' && rowData['district_name'] === 'All' && rowData['block_name'] === 'All'
```

Comment:
When will this condition be satisfied when all of them will be 'All'?

Reply:
yes. For "All" row backend is sending all the location names as "All"

Reply:
I see